src/components/build/userProfilePage/UserProfileMenu.tsx

```diff
@@ -55,7 +55,7 @@ class UserProfileMenu extends Component<UserMenuProps, UserMenuState> {
         <PageHeader
           searchPlaceholder="Search by Name, Email or Subject"
           search={() => {}}
-          searching={(v) => {}}
+          searching={(v: string) => {}}
```

Comment:
@william-gooch  v is incoming value. Type should be defined in the PageHeader component. there is no need to define it here